Transition MangaBox to 0.8.

Batmeow opened the pull request:
Match functional parity with the MangaBox 0.6 Sources but with the 0.8 Source format.

Batmeow committed:
Transition MangaBox to 0.8.

niclimcy commented:
Missing EOL on all files, otherwise looks good!

TheNetsky commented on src/MangaKakalot/MangaKakalot.ts:
```diff
@@ -0,0 +1,172 @@
+            type: BadgeColor.GREEN
+        }
+    ],
+    intents: SourceIntents.HOMEPAGE_SECTIONS | SourceIntents.MANGA_CHAPTERS
```
Does Mangakakalot not have a second image server? Pretty sure it does?

Batmeow replied:
It does but the value it uses to determine which server to use is determined by a generated cookie. But then there are times when MangaKakalot redirects to the Manganato (chapmanganato) reader which in those cases the image server settings would match what Manganato & MangaBat use. I'll re-enable the setting for MangaKakalot so it can be used in those senarios.

TheNetsky commented on src/MangaBoxParser.ts:
```diff
@@ -0,0 +1,208 @@
+            id: id,
+            chapNum: isNaN(chapNum) ? 0 : chapNum,
+            name: name,
+            group: 'Aggragator',
```
You can just put an empty string here if not being used.

Batmeow replied:
Ah ok, figured it might look empty without something there. I'll make the changes.

TheNetsky commented on src/MangaBoxParser.ts:
```diff
@@ -0,0 +1,208 @@
+            status: status,
+            author: author ? author : 'Unkown',
+            desc: desc,
+            hentai: hentai,
```
I don't think the hentai property is required/being used anymore

Batmeow replied:
The type definition showed it as optional so wasn't sure. I'll remove it if it's no longer being used.

Batmeow committed:
Fixes.
- Add MangaKakalot Source Settings for Image Server.
- Remove the Hentai optional bool on parseMangaDetail.
- Update LangCode.

Batmeow committed:
Add EOL on all files.
